src/components/attendee/AttendeeLayout/AttendeeLayout.tsx

```diff
@@ -70,8 +70,8 @@ export const AttendeeLayout: React.FC<AttendeeLayoutProps> = ({ space }) => {
             <VenuePage setBackButtonSpace={setBackButtonSpace} />
           </MainSection>
           <div className={layerUIClasses}>
-            <ChatContainer isRelative={isChatRelative} />
-            <VideoHuddle />
+            <ChatContainer isRelative={isFooterRendered} />
+            <VideoHuddle isFooterRendered={isFooterRendered} />
```

Comment:
`isFooterRendered` is slightly missleading as a prop name and makes me think that the `VideoHuddle` component has it's own footer. Maybe change the VideoHuddle prop name to something like `isFixed`, `fixed` , `fixedPosition` and then `={isFooterRendered}`

Reply:
fixed [2da2eda](https://github.com/sparkletown/sparkle/pull/3020/commits/2da2eda73e5e2f280f1723de03ea890e90bf06e3)